src/data/nomad.json

```diff
@@ -94,6 +94,11 @@
 				"icon": "api",
 				"pathSuffix": "api-docs"
 			},
+			{
+				"label": "Intro",
+				"icon": "docs",
+				"pathSuffix": "intro"
```

Comment:
`/nomad/intro` path was missing from product config, have added it in to fix. Likely something that needs discussion, but outside the scope of this PR (or rather, exposing this type of thing more broadly in preview links is the purpose of this PR). See `Anything else` section in PR description for details.